.travis.yml

```diff
@@ -32,21 +32,10 @@ install:
   - pip install -e .[users] .[qa]
 
 before_script:
-  - ./runflake8
-  - ./runisort
-  - ./runcheckmigration
-  - ./runcheckendline
+  - runchecks --migration-path "./tests/test_project/migrations"
```

Comment:
let's rename this script: `openwisp-utils-qa-checks`, this way we will make it obvious where the script comes from and contributors will quickly be able to find it